### ISSUE TITLE
[Keras:Metrics] Use the val prefix for validation metrics

### ISSUE THREAD

**Issue by popojargo** [PhilipMay/mltb#17]
I've recently used your package and I was misleaded by the metric name. Usually in keras, validation metrics are prefixed with `val`. Maybe it should be better idea to use `val_f1` and `val_roc_auc` since it's evaluated against validation set.

Also thanks for the library!

**Comment by PhilipMay** [comment]
Yes you are right. Good point.
Are you willing to provide a PR?

**Comment by popojargo** [comment]
I could. I guess we keep the default `f1` and `roc_auc` and it could be removed when doing a major release?

**Comment by PhilipMay** [comment]
Nah. You can just do the change. There was no real major release yet.
A PR would be great. Many thanks.
